packages/cloudvision-grpc-web/generated/arista/subscriptions/subscriptions.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable */
+import Long from 'long';
```

Comment:
I though Long was getting removed?

Reply:
It'll be a while before the protobufjs fork is merged and then ts-proto will be updated to support it. If we're generating these files from the vanilla implementation, this will have to be here.